gem/templates/profiles/viewprofile.html

```diff
@@ -14,27 +14,23 @@
         <hr>
         <div class="nickname-form">
             <div class="block">
-                <form method="post" id="frmProfileEdit"
-                      action="{% url 'molo.profiles:edit_my_profile' %}">
-                    {% csrf_token %}
-                    <h5>{% trans "Change your name" %}</h5>
-                    <p>{% trans "When you type comments your name will appear as:" %}
-                    <span>{{ user.profile.alias }}</span></p>
 
-                    <p>{% trans "Please do not use your real name, phone number or email address as it's unsafe." %}</p>
```

Comment:
@codiebeulaine I meant you're removing it here (but I see you added it to the edit page)